src/koga/configure.lisp

```diff
@@ -513,10 +513,21 @@
         :initform nil
         :type (or null pathname)
         :documentation "The git binary to use.")
+   (lisp :accessor lisp
+         :initarg :lisp
+         :initform #+abcl #P"abcl"
+                   #+ccl #P"ccl"
+                   #+clasp #P"clasp"
+                   #+clisp #P"clisp"
+                   #+ecl #P"ecl"
+                   #+sbcl #P"sbcl"
+                   #-(or abcl ccl clasp clisp ecl sbcl) nil
```

Comment:
if we load on some alien lisp, will NIL result in anything sensible later on? If I'm reading lisp-command correctly, it might end up trying to run a program called "NIL". if the results are weird, we should instead make the initform an error, or otherwise make a sensible error pop up.

Reply:
Thats a good point